src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3344,6 +3347,21 @@
     <value>(multiple values)</value>
     <comment>Text indicating that multiple selected files have different metadata values.</comment>
   </data>
+  <data name="Author" xml:space="preserve">
+    <value>Author</value>
+  </data>
+  <data name="GitDateCommitted" xml:space="preserve">
```

Comment:
```suggestion
  <data name="DateCommitted" xml:space="preserve">
```
The name should match the content, the `Git` is extra because it's not part of the string.